tools/jenkins/jenkins.sh

```diff
@@ -45,6 +45,9 @@ build_sqoop
 
 make apps
 
+build/env/bin/pip install pylint
```

Comment:
Thanks!

Just if there is already a pylint, shound't we skip this install?

Reply:
Okay. I write an additional commit.